README.md

```diff
@@ -177,6 +177,7 @@
 
 ### Courses
 
+- [Modern APIs with FastAPI and Python](https://training.talkpython.fm/courses/getting-started-with-fastapi)
```

Comment:
```suggestion
- [Modern APIs with FastAPI and Python](https://training.talkpython.fm/courses/getting-started-with-fastapi) - A course designed to get you creating new APIs running in the cloud with FastAPIs quickly. 
```

Reply:
The text is copied from the page, but `FastAPIs` should be `FastAPI` :sweat_smile:

Reply:
Right, I missed the description. 👍